src/components/Proposal/Voting/VoteButtons.tsx

```diff
@@ -86,24 +86,22 @@ class VoteButtons extends React.Component<IContainerProps, IState> {
                             proposal.stage === IProposalStage.Executed ||
                             (proposal.stage === IProposalStage.Boosted && expired) ||
                             (proposal.stage === IProposalStage.QuietEndingPeriod && expired) ||
-                            !currentAccountState ||
-                            currentAccountState.reputation.eq(new BN(0)) ||
+                            (currentAccountState &&
+                            currentAccountState.reputation.eq(new BN(0))) ||
                             !!currentVote;
 
     const tipContent = (vote: IProposalOutcome): string =>
-      !currentAccountState ?
-        "Cannot vote - please log in" :
-        currentVote ?
-          "Can't change your vote" :
-          currentAccountState.reputation.eq(new BN(0)) ?
-            "Voting requires reputation in " + dao.name :
-            proposal.stage === IProposalStage.ExpiredInQueue || (proposal.stage === IProposalStage.Boosted && expired) || (proposal.stage === IProposalStage.QuietEndingPeriod && expired) ?
-              "Can't vote on expired proposals" :
-              proposal.stage === IProposalStage.Executed ?
-                "Can't vote on executed proposals" :
-                isVoting ?
-                  "Warning: Voting for this proposal is already in progress" :
-                  `Vote ${vote === IProposalOutcome.Pass ? "for" : "against"}`
+      currentVote ?
+        "Can't change your vote" :
+        currentAccountState.reputation.eq(new BN(0)) ?
```

Comment:
wait so there will always be a currentAccountState now?

Reply:
good catch